src/views/colours.njk

```diff
@@ -0,0 +1,54 @@
+---
+title: Colours
+description: This colour palette should be used to help services look and feel consistent
+layout: default
+eleventyNavigation:
+  key: Colours
+  section: styles
+---
+
+{% macro swatch(key, colour) %}
+    {% if colour == "#FFFFFF" %}
+        {% set border = "border border-grey-4" %}
+    {% endif %}
+    <div class="flex items-center mb-4">
+        <div class="text-white w-16 h-16 rounded-full mr-4 {{border}}" style="background-color:{{colour}}"></div>
+        <div>
+            <div class="font-mono font-bold">${{key}}</div>
+            <div class="font-mono text-sm text-grey-2">{{colour}}</div>
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro section(name, description, palette) %}
+    <section class="mb-2">
+        <h2 class="mb-4">{{name}}</h2>
+        <p>{{description}}</p>
+        <div class="grid grid-cols-2 md:grid-cols-3 py-4">
+            {% for key, colour in palette %}
+                {{swatch(key, colour)}}
+            {% endfor %}
+        </div>
+    </section>
+{% endmacro %}
+
+<header class="mb-8">
+    <div class="text-grey-2 text-lg mb-1">Styles</div>
+    <h1 class="mb-6">{{title}}</h1>
+    <p>This colour palette should be used to help services look and feel consistent.</p>
+    <p>To use the palette, <a href="https://sass-lang.com/guide#topic-2" rel="external">Sass variables</a> are provided and should be used wherever possible. Using the raw hexadecimal (hex) values (eg. <code>#206095</code>) should be avoided to ensure a <a href="single source of truth" rel="external">single source of truth</a>—making updating and maintaining colours easier.</p>
+</header>
+
+{{section("Main", "This palette is for general use.", colours.general)}}
+{{section("Supporting", "Supporting colours are provided to supplement the main colours.", colours.supporting)}}
```

Comment:
Is there any specific context around why/when to use the supporting colour palette? If so, perhaps we can include it here

Reply:
Not yet, I didn't want to just make stuff up